pyproject.toml

```diff
@@ -7,12 +7,13 @@ repository = "https://github.com/eliorc/node2vec"
 license = "MIT"
 
 [tool.poetry.dependencies]
-python = "^3.8"
-networkx = "^2.5"
+python = "^3.11"
```

Comment:
Python `^3.10` is actually sufficient.

Reply:
@Crosswind what is the minimum Python version to support `gensim==4.3.3`? I belive it's best to go with that

Reply:
`gensim==4.3.3` works with Python 3.8.